client/components/Cart.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Icon } from 'react-materialize';
+import { connect } from 'react-redux';
+import { removeItem } from '../store/cart';
+
+function Cart(props) {
+  return (
+    <div className="container">
```

Comment:
It's a little jarring to see this file switch away from `react-materialize` components for layout such as `<Row>`, `<Collection>`, using `<div className="row center">` instead.

Upon investigation I see `react-materialize` doesn't provide as many components to wrap tables. So that's a little easier to understand.